testnet-contracts/scripts/mintTestTokens.js

```diff
@@ -21,33 +21,42 @@ module.exports = async () => {
   const NUM_ARGS = process.argv.length - 4;
 
   // Mint transaction parameters
-  const TOKEN_AMOUNT = 1000;
-
+  const TOKEN_AMOUNT = (1).toString().padEnd(20, "0")
+    console.log({TOKEN_AMOUNT})
```

Comment:
```suggestion
```

Reply:
I think it's actually good to have this number logged during the command